insecure/integration/functional/test/gossipsub/scoring/scoring_test.go

```diff
@@ -115,7 +116,9 @@ func testGossipSubInvalidMessageDeliveryScoring(t *testing.T, spamMsgFactory fun
 		t.Name(),
 		idProvider,
 		p2ptest.WithRole(role),
-		p2ptest.OverrideFlowConfig(cfg))
+		p2ptest.OverrideFlowConfig(cfg),
+		p2ptest.WithValidateQueueSize(3000), // prevent node from dropping messages on slow CI machines
```

Comment:
the invalid signature test seems to run slower on the new version. Added this to prevent the victim node from dropping too many messages during the test.